src/main/scala/Game.scala

```diff
@@ -0,0 +1,91 @@
+class Game {
+  var player1: Player = _
+  var player2: Player = _
+
+  def startGame(): Unit = {
+    menu()
+
+    val roundsStr = scala.io.StdIn.readLine("How many rounds? ")
+    val rounds = roundsStr.toInt
+
+    do {
+      // Store the players moves - Ai will make decisions based on opposition moves
+      player1.getOppositionMoves = player2.moveList
```

Comment:
Why not just pass each individual move rather than throwing the entire list each time?